spec/rails_helper.rb

```diff
@@ -28,17 +30,30 @@
 # If you are not using ActiveRecord, you can remove this line.
 ActiveRecord::Migration.maintain_test_schema!
 
+Capybara.register_driver :selenium do |app|
+ Capybara::Selenium::Driver.new(
+   app,
+   browser: :firefox,
+   desired_capabilities: Selenium::WebDriver::Remote::Capabilities.firefox(marionette: false)
+ )
+end
+
 RSpec.configure do |config|
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
   config.fixture_path = "#{::Rails.root}/spec/fixtures"
 
   # If you're not using ActiveRecord, or you'd prefer not to run each of your
   # examples within a transaction, remove the following line or assign false
   # instead of true.
-  config.use_transactional_fixtures = true
+  #
+  # Selenium, which is a defult driver for specs using js, doesn't support transactional_fixtures
+  # Remember to add "js: true" to your spec's "describe" method
```

Comment:
Please add this to the `README` - how to debug capybara tests.
defult -> default